test/configCases/output/function/webpack.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+	entry() {
+		return {
+			a: "./a"
+		};
+	},
+	output: {
+		filename: () => "[name].js"
```

Comment:
Could you somehow use the provided argument?

Like ``filename: data => `${data.chunk.name}.js` ``